docs/wiki/resources.md

```diff
@@ -2,7 +2,7 @@
 
 ## More Documents
 
-    [Build and run automation](extra-documents/build-run-automation.md) </br>
+:   [Build and run automation](extra-documents/build-run-automation.md) </br>
```

Comment:
What is the point of these added colons?

Reply:
Without colons, links are not rendered correctly. Here **Other** section has colon, while **Talks** and **More Documents** missing it. 

<img width="847" alt="resim" src="https://github.com/user-attachments/assets/be64d6de-0746-41b2-80f7-269d9d6a0f01" />


Here the resources.md missing colons, only Other have it.

<img width="913" alt="resim" src="https://github.com/user-attachments/assets/5c9134d4-fb28-4566-947b-4d5e68c70416" />

Reply:
Maybe that's because we're mixing markdown and html [1] ?
I'd like to see a proper fix. Can we use some markdown instead of the </br> ?

[1] https://stackoverflow.com/questions/64201516/github-markdown-links-in-list-do-not-display-correctly

Reply:
enumerated all list items as proposed in stackoverflow answer.